check_synology.py

```diff
@@ -161,7 +171,7 @@ def exitCode():
         state = 'CRITICAL'
 
     update_status = status_translation.get(update_status_nr)
-    print state + ' - DSM Version: %s, DSM Update: %s' % (update_dsm_verison, update_status), '| DSMupdate=%sc' % update_status_nr
+    print(state + ' - DSM Version: %s, DSM Update: %s' % (update_dsm_verison, update_status), '| DSMupdate=%sc' % update_status_nr)
```

Comment:
This now results in:
```(u'OK - DSM Version: DSM 6.2-24922, DSM Update: Unavailable', u'| DSMupdate=2c')``` instead of
```OK - DSM Version: DSM 6.2-24922, DSM Update: Unavailable | DSMupdate=2c```
I think once again the parentheses are the problem

Reply:
Hmm, python3 needs the parentheses (will not print them!), while python2 will print them verbatim.
See e.g.http://python3porting.com/noconv.html, https://python-future.org/compatible_idioms.html . so if you can live with python2.6+, then you could import __future__ and make it work with python2 and python3.

Reply:
As python2 is deprecated I'd opt for only supporting python3 and and state version 0.1 as the last python2 compatible release.

Reply:
Hi there,

having those improvements integrated into mainline would be awesome to save resources. I also would like to express my support for dropping Python 2 compatibility here. Do you have any strong objections against it, @wernerfred?

With kind regards,
Andreas.

Reply:
No objections from my side. This PR is outdated and needs to be rebased with conflicts. (#8 already introduced python3 support). So missing changes are only switching from pysnmp to easysnmp.
Would you have time to solve the conflicts and update the PR?

Reply:
Hi Frederic,

we just submitted #19. For testing purposes, we are also running this in production now. While we have not come across a timeout situation (#17), we believe it works well so far and would be ready to be merged.

Maybe you want to tag 0.3.1 beforehand, in order to mark this as the most recent release based on `pysnmp`?

With kind regards,
Andreas.

Reply:
> Maybe you want to tag 0.3.1 beforehand, in order to mark this as the most recent release based on `pysnmp`?

Good point. 
The easysnmp pr #19 will be a breaking change and qualify for `1.0.0`

Reply:
Thank you, I've added f2e973ad3. I think it will be safe to close this PR. Thanks again for your contribution, @kainhofer!